<launch>

    <!-- Arguments -->
    <arg name="namespace" default="stereo"/>
    <arg name="env" default="real"/>

    <!-- Run the stereo image proc -->
    <node ns="$(arg namespace)" pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" output="screen">
        <remap from:="front_left_camera/image_raw" to:="stereo/left/image_raw" />
        <remap from:="front_left_camera/camera_info" to:="stereo/left/camera_info" />
        <remap from:="front_right_camera/image_raw" to:="stereo/right/image_raw" />
        <remap from:="front_right_camera/camera_info" to:="stereo/right/camera_info" />
        <param name="approximate_sync" value="True"/>

        <param name="prefilter_size" value="5"/>
        <param name="prefilter_cap" value="54"/>
        <param name="correlation_window_size" value="9"/>
        <param name="min_disparity" value="59"/>
        <param name="disparity_range" value="32"/>
        <param name="uniqueness_ratio" value="5.0"/>
        <param name="texture_threshold" value="20"/>
        <param name="speckle_size" value="5"/>
        <param name="speckle_range" value="15"/>
        v

    </node>

</launch>
